account_product_fiscal_classification/__manifest__.py

```diff
@@ -7,36 +6,22 @@
 {
     'name': 'Account Product - Fiscal Classification',
     'summary': 'Simplify taxes management for products',
-    'version': '10.0.1.0.0',
+    'version': '12.0.1.0.0',
     'category': 'Accounting',
     'author': 'Akretion,GRAP,La Louve,Odoo Community Association (OCA)',
-    'website': 'https://odoo-community.org/',
+    'website': 'https://github.com/OCA/account-fiscal-rule',
     'license': 'AGPL-3',
     'depends': [
         'account',
-        'l10n_generic_coa',
     ],
     'data': [
         'security/ir_rule.xml',
-        'security/ir_model_access.yml',
+        'security/ir.model.access.csv',
         'views/view_related.xml',
         'views/action.xml',
         'views/view.xml',
         'views/menu.xml',
         'views/view_product_category.xml',
     ],
-    'demo': [
-        'demo/account_tax.yml',
-        'demo/account_tax_template.yml',
-        'demo/account_product_fiscal_classification.yml',
-        'demo/account_product_fiscal_classification_template.yml',
-        'demo/product_category.xml',
-        'demo/product_template.yml',
-        'demo/res_company.yml',
-        'demo/res_groups.yml',
-        'demo/res_users.yml',
-    ],
-    'post_init_hook':
-        'create_fiscal_classification_from_product_template',
```

Comment:
Hi @PierrickBrun. Thanks for porting this module. Don't understand why you removed this call.
How do you populate the database, when you install the module on a db with existing products ? 
regards.

Reply:
and please, dont mark comment as solved if not solved ;-)

![image](https://user-images.githubusercontent.com/3407482/60987152-4c4fa300-a341-11e9-841a-45fa00e07cae.png)